infra/lib/fhir-server-stack.ts

```diff
@@ -55,10 +57,12 @@ export function settings(): Settings {
       ...defaults,
       cpu: 4 * vCPU,
       memoryLimitMiB: 8192,
-      taskCountMin: 6,
-      taskCountMax: 10,
-      minDBCap: 10,
-      maxDBCap: 48,
+      taskCountMin: 8,
+      taskCountMax: 20,
+      minDBCap: 15,
+      maxDBCap: 96,
+      thresholdVolumeReadIops: 2_500_000,
+      thresholdVolumeWriteIops: 2_500_000,
```

Comment:
It's been under 60% ACU w/ these under 2.5MM IOPS - [source](https://us-west-1.console.aws.amazon.com/rds/home?region=us-west-1#database:id=fhir-server;is-cluster=true;tab=monitoring)